### PR TITLE
Akka.Router: automatically unpack `IScheduledTellMsg` when being handled through a router

### PR DESCRIPTION
**Pull request by Aaronontheweb**
## Changes

Working on reproducing #7247

## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).
* [x] Design discussion issue #7247

### CODE REVIEW

File: src/core/Akka.Tests/Actor/Scheduler/BugFix7247Spec.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        var noCellRef = Sys.As<ExtendedActorSystem>().Provider.CreateFutureRef(tcs);
+        var router = Sys.ActorOf(Props.Empty.WithRouter(new BroadcastGroup()));
+        var routee = Routee.FromActorRef(noCellRef);
+        router.Tell(new AddRoutee(routee));
+
+        await AwaitAssertAsync(async () =>
+        {
+            var allRoutees = await router.Ask<Routees>(GetRoutees.Instance);
+            allRoutees.Members.Count().ShouldBeGreaterThan(0);
+        });
```
Comment (Aaronontheweb) [inline src/core/Akka.Tests/Actor/Scheduler/BugFix7247Spec.cs:55]: You can't target a `/temp` actor via `ActorSelection`, so I had to go through some extra steps in order to get the `FutureActorRef` added to the `BroadcastGroup` router here.

File: src/core/Akka/Routing/RoutedActorCell.cs [hunk 2]
```diff
@@ -193,7 +194,14 @@ public override void SendMessage(Envelope envelope)
             if (RouterConfig.IsManagementMessage(envelope.Message))
                 base.SendMessage(envelope);
             else
-                Router.Route(envelope.Message, envelope.Sender);
+            {
+                // Bugfix: https://github.com/akkadotnet/akka.net/issues/7247 
+                if(envelope.Message is IScheduledTellMsg scheduledTellMsg)
```
Comment (Aaronontheweb) [inline src/core/Akka/Routing/RoutedActorCell.cs:199]: This is the real fix here - and I describe the root of problems here on the issue itself https://github.com/akkadotnet/akka.net/issues/7247#issuecomment-2162089219

Comment (Aaronontheweb) [reply src/core/Akka/Routing/RoutedActorCell.cs:199]: Technically this change means that certain messages will now show up in Phobos traces if they're sent through a router via the scheduler. I am fine with that.

**Review by Aaronontheweb — COMMENTED** [review]
Detailed my changes

**Review by Arkatufus — APPROVED** [review]
LGTM

**Pull request merged**